README.md

```diff
@@ -1,6 +1,16 @@
 [![Documentation Status](https://readthedocs.org/projects/qtt/badge/?version=latest)](https://qtt.readthedocs.io/en/latest/?badge=latest)
 [![PyPI version](https://badge.fury.io/py/qtt.svg)](https://badge.fury.io/py/qtt)
 
+# Important notice
+
+As of December 2023, this repository is no longer actively maintained. This means the maintenance team:
+* will not release any new version. The latest version 1.4.0 will remain on pypi
+* will not take action on open or any new issues, questions or feature requests
```

Comment:
This is redundant, as for an archived repo no new issues or PRs can be opened.

Reply:
Will change it to: will not take action on currently open issues, merge requests etc